plugin/adsk/scripts/mayaUsd_layerEditorFileDialogs.mel

```diff
@@ -42,12 +61,25 @@ global proc string UsdLayerEditor_SaveLayerFileDialog()
         $startDir = dirname($fullPath);
     }
 
+    string $createCallback;
+    string $initCallback;
+    string $commitCallback;
+    if ($isRootLayer) {
+        $createCallback = "mayaUsd_USDRootFileRelative_UICreate";
+        $initCallback   = "mayaUsd_USDRootFileRelative_UIInit";
+        $commitCallback = "mayaUsd_USDRootFileRelative_UICommit";
+    } else {
+        $createCallback = "mayaUsd_USDSubLayerFileRelative_UICreate";
+        $initCallback   = "mayaUsd_USDSubLayerFileRelative_UIInit";
+        $commitCallback = "mayaUsd_USDSubLayerFileRelative_UICommit";
```

Comment:
You'll need to update this with the fileTypeChanged/selectionChanged callbacks which I just added. So you'll need two new MEL procs for your SubLayer case.